### PR TITLE
fix: use strconv.ParseInt instead of strconv.Atoi

### PR DESCRIPTION
**Pull request by brenkem**
Regarding:
https://github.com/kha7iq/pingme/issues/47

### CODE REVIEW

File: service/telegram/telegram.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ All configuration options are also available via environment variables.`,
 				if len(v) <= 0 {
 					return helpers.ErrChannel
 				}
-				k, errStr := strconv.Atoi(v)
+				k, errStr := strconv.ParseInt(v, 10, 64)
 				if errStr != nil {
 					return errStr
 				}
```
Comment (kha7iq) [inline service/telegram/telegram.go:84]: Now that its already int64 can you please change the AddReceivers to 
```
telegramSvc.AddReceivers(k)
```

**Review by kha7iq — CHANGES_REQUESTED** [review]
Can you please make the minor change.
Thank you

File: service/telegram/telegram.go (hunk 1 above)
Comment (brenkem) [reply service/telegram/telegram.go:84]: Okay, I fixed the type cast too. See second commit.

**Comment by kha7iq** [comment]
Thank you @brenkem for taking the time to make the changes.
Everything is good to merge now, you have not set the username for your github when you made the commits.
You can do so by setting it this way.
```
git config --global user.name "yourusername"
git config --global user.email "githubaccount@email.com"
```
Although its not an issue and i can merge, i will be happy if you get the recognition for your contributions.
Do let me know if you would like to rebase and fix the commits, or should i continue to merge.

**Comment by brenkem** [comment]
I do only work under my account name. No need to change anything here. You can go and merge the change. No extra work needed here.

Thanks for your nice tool, have fun.

**Comment by kha7iq** [comment]
Well noted, and thanks once again

**Pull request merged**